libsolidity/ast/ASTJsonConverter.h

```diff
@@ -148,15 +150,22 @@ class ASTJsonConverter: public ASTConstVisitor
 		return _node.id();
 	}
 	template<class Container>
-	static Json::Value getContainerIds(Container const& container)
+	static Json::Value getContainerIds(Container const& _container)
```

Comment:
I think there was another use of this template function where the order did matter, please check.

Reply:
Ah right. The linearizedContractDependencies order matters. fixed